src/documents/tutorial-developerIntroduction/DeveloperIntroductionToInfusionFramework-Components.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Developer Introduction to the Infusion Framework - Components
+layout: default
+category: Tutorials
+---
+
+Programs using Infusion structure their data, functionality and interaction as various components defined using a common syntax.
+
+Components are [JavaScript objects that have certain characteristics](/infusion/development/UnderstandingInfusionComponents.md). Components can be freely combined together to form new components using the [grades system](/infusion/development/ComponentGrades.md), or organized into relationships with one another via [subcomponents](/infusion/development/SubcomponentDeclaration.md).
+
+ There are three base components to be aware of that any new components you create with Infusion will likely have at their root:
+
+ * [`fluid.component`](/infusion/development/ComponentConfigurationOptions.md#options-supported-by-all-components-grades), the simplest type with support for events and public function definition (called [invokers](/infusion/development/Invokers.md) in Infusion).
```

Comment:
I will stop pointing out the broken links from this point on, but wanted to comment that even though this link should work once we deploy, it should be possible to also test your links as part of the review.